Cargo.toml

```diff
@@ -44,12 +44,13 @@ iced-x86 = { version = "1.17.0", default-features = false, features = [
     "gas",
 ] }
 itertools = "0.14.0"
+leb128 = "0.2.5"
 libc = "0.2.152"
 memchr = "2.6.0"
 memmap2 = "0.9.0"
 mimalloc = { version = "0.1", default-features = false }
 normalize-path = "0.2.0"
-object = { version = "0.36.7", default-features = false, features = [
+object = { git = "https://github.com/gimli-rs/object", rev = "61474b7de8074f91b78ad24ff2424e54a22f8b32", default-features = false, features = [
```

Comment:
I guess it's a good thing that we just did a release of wild, since our next release will need to wait until after this is resolved :)

Reply:
Yeah! I've created #778 to track the thing. If we really need to do a release, we can still switch to a proper release and add a couple of newly added constants.